### PR TITLE
Feature/lu 5452 egest pipeline

### PR DESCRIPTION
**Pull request by larissanne**

### CODE REVIEW

File: concoursePipeline/egestPipeline.yml [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+  - name: base-go-image
+    type: docker-image
+    source:
+      repository: golang
+      tag: 1.13.4-stretch
+
+  - name: ecr-image-egest
+    type: docker-image
+    source:
+      repository: 226575302242.dkr.ecr.eu-west-2.amazonaws.com/takeon-validation-cicd
```
Comment (RyanAldy) [inline concoursePipeline/egestPipeline.yml:23]: Parameter

File: concoursePipeline/egestPipeline.yml [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+      access_token: ((aws.git_access_token))
+
+  - name: egest-deploy-repository
+    type: git
+    check_every: 24h
+    source:
+      uri: https://github.com/ONSdigital/takeon-egest-pipeline.git
+      paths: 
+        - takeon-dbexport/
+      branch: dev
```
Comment (RyanAldy) [inline concoursePipeline/egestPipeline.yml:42]: parameter

File: concoursePipeline/egestPipeline.yml [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+  
+  - name: takeon-egest-pipeline-git-repo
+    type: git
+    source:
+      uri: https://github.com/ONSdigital/takeon-egest-pipeline.git
```
Comment (RyanAldy) [inline concoursePipeline/egestPipeline.yml:47]: parameter

File: concoursePipeline/egestPipeline.yml [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+      paths:
+        - aws-infrastructure/
+      branch: feature/LU-5452-egestPipeline
```
Comment (RyanAldy) [inline concoursePipeline/egestPipeline.yml:50]: parameter

File: concoursePipeline/egestPipeline.yml [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+    plan:
+      - get: egest-deploy-repository
+        trigger: true
+      - task: serverless-aws
+        config:
+          platform: linux
+          image_resource:
+            type: docker-image
+            source:
+              repository: 226575302242.dkr.ecr.eu-west-2.amazonaws.com/takeon-validation-cicd
```
Comment (RyanAldy) [inline concoursePipeline/egestPipeline.yml:137]: Parameter

File: takeon-dbexport/Dockerfile [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+RUN apt-get update -y &&\
+    apt-get install awscli -y &&\
+    curl -sL https://deb.nodesource.com/setup_10.x | bash &&\
+    apt-get install nodejs -y &&\
+    node -v &&\
+    npm -v &&\
+    npm install -g serverless@1.52.2
+
+
+# FROM debian:stable-slim
```
Comment (RyanAldy) [inline takeon-dbexport/Dockerfile:20]: Remove the commented code

File: takeon-dbexport/serverless.yml [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+# You can always add more config options for more control.
+# We've included some commented out config examples here.
+# Just uncomment any of them to get that config option.
+#
+# For full config options, check the docs:
+#    docs.serverless.com
+#
+# Happy Coding!
+
+service: takeon-db-export-lambda-${var.user} # NOTE: update this with your service name
```
Comment (RyanAldy) [inline takeon-dbexport/serverless.yml:14]: Not sure if the variable can be substituted in like this.  Will have to go in the serverless-config.json and extracted from there

File: takeon-dbexport/serverless.yml [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+# You can pin your service to only deploy with a specific Serverless version
+# Check out our docs for more details
+# frameworkVersion: "=X.X.X"
+frameworkVersion: ">=1.28.0 <2.0.0"
+
+provider:
+  name: aws
+  runtime: go1.x
+  region: eu-west-2
+  deploymentBucket: takeon.validation.sandbox.deployment
```
Comment (RyanAldy) [inline takeon-dbexport/serverless.yml:27]: Should we parameterise this with user too?

File: takeon-dbexport/serverless.yml (hunk 7 above)
Comment (mattprice-28) [reply takeon-dbexport/serverless.yml:14]: This is terraform notation isn't it? I think you could do {{ user }} in yml

**Pull request merged**